Update readme to include migration guide for users on the 0.x helm chart (#197)
superblocksteam/agent: README.md

helm upgrade -i -n superblocks superblocks-agent superblocks/superblocks-agent \
  --create-namespace \
  --set superblocks.agentKey='<agent-key>' # obtained during agent onboarding \
  --set superblocks.agentHostUrl='http[s]://<agent-host[:port]>' \
  --set superblocks.agentTags='<"profile:*"|"profile:staging"|"profile:production">' \
  --set superblocks.agentDataDomain='<"app.superblocks.com"|"eu.superblocks.com">'
```

#### v0.x to v1.0 Migration Guide

* Container ports are now specified under `service.ports.http`. It is recommended that you leave these settings as default.

> * Agent URL field `superblocks.agentUrl` must be updated from `https://myagent.example.com/agent` to `https://myagent.example.com`. The new OPA no longer has the `agent` prefix to its endpoints.

Reviewer: ```suggestion
* Agent URL field `superblocks.agentUrl` must be updated from `https://myagent.example.com/agent` to `https://myagent.example.com`. The new OPA no longer has the `agent` suffix path on its endpoints.
```

* Agent Environment variable `superblocks.agentEnvironment` is deprecated and you must start using `superblocks.agentTags`.
  * `superblocks.agentEnvironment: "*"` is equivalent to `superblocks.agentTags: "profile:*"`
  * `superblocks.agentEnvironment: "staging"` is equivalent to `superblocks.agentTags: "profile:staging"`
  * `superblocks.agentEnvironment: "production"` is equivalent to `superblocks.agentTags: "profile:production"`

> * The `controller` block in the values file is deprecated. Everything underneath can be specified at one level higher. You can simply delete the `controller` block and unindent everything underneath it.

Reviewer: ```suggestion
* The `controller` block in the values file is deprecated. Everything underneath can be specified one level higher. You can simply delete the `controller` block and unindent everything underneath it.
```

```sh
# obtained during agent onboarding
export SUPERBLOCKS_AGENT_KEY='<agent-key>'

> export SUPERBLOCKS_AGENT_HOST_URL='http[s]://<agent-host[:port]>'

Reviewer: Do we still need to call out the port? Or does the syntax mean it's optional?

Author: Yeah, the syntax makes it optional. It's only needed if it's not on the default http/https ports
